contributor/06-conclusion-article.asciidoc

```diff
@@ -1,18 +1,18 @@
 == Conclusion
 
-Thanks for watching the Contributor segment of the InnerSource Commons Learning Path. With this section you've learned about the Contributor role - the life blood of InnerSource projects: External to the owners of a component's team, they bring additional valuable input to the project.
+Thanks for reviewing the Contributor segment of the InnerSource Commons Learning Path. With this section you've learned about the Contributor role - the life blood of InnerSource projects: external to the component owners' team, they bring additional valuable input to the project.
 
-In this section you've learned about how to become an contributor by learning about how to find opportunities to contribute and the mindset and habits needed to find or create such opportunities.
-We've also discussed the ethos of the role and suggested some behaviour that is likely to lead to successful contributions.
+In this section you've learned about how to become a contributor by finding opportunities to contribute, and the mindset and habits needed to find or create such opportunities.
```

Comment:
This is a list rather than a compound sentence, so no comma needed.

```suggestion
In this section you've learned about how to become a contributor by finding opportunities to contribute and the mindset and habits needed to find or create such opportunities.
```

Reply:
Was this meant to say we are finding the mindset and habits (along with opportunities)?
Or are we learning the mindset and habit (along with how to become a contributor)?
I feel it meant to convey the latter, but without the comma it reads as the former.
Anyway, I can remove the comma and let it be, or can we rephrase it in a less ambiguous way?
Thank you!!!

Reply:
I guess the intended meaning was to summarize the individual sections, two of which should be referred to in this sentence. 
Without knowing that context it might indeed read a bit ambiguous. How about proposing a commit suggestion for an alternative phrasing?

Reply:
Maybe we should split into two sentences so the meaning is clear.

Reply:
How about now?

Reply:
I like the new change.